src/pages/Staking/AuthorizeStakingApps/AuthorizeApplicationsCardCheckbox/index.tsx

```diff
@@ -177,40 +270,122 @@ export const AuthorizeApplicationsCardCheckbox: FC<
-        <GridItem gridArea="token-amount-form" mt={5}>
-          <TokenAmountForm
-            innerRef={formRef}
-            onSubmitForm={onAuthorizeApp}
-            label="Amount"
-            submitButtonText={
-              appAuthData.isAuthorized
-                ? `Authorize Increase`
-                : `Authorize ${appAuthData.label}`
-            }
-            maxTokenAmount={maxAuthAmount}
-            placeholder={"Enter amount"}
-            minTokenAmount={
-              appAuthData.percentage === 0
-                ? minAuthAmount
-                : WeiPerEther.toString()
-            }
-            helperText={`Minimum ${formatTokenAmount(minAuthAmount)} T for ${
-              appAuthData.label
-            }`}
-          />
-        </GridItem>
+        {!hasPendingDeauthorization && (
+          <GridItem gridArea="token-amount-form" mt={5}>
+            <TokenAmountForm
+              innerRef={formRef}
+              onSubmitForm={onSubmitForm}
+              label="Amount"
+              submitButtonText={
+                isIncreaseAction
+                  ? appAuthData.isAuthorized
+                    ? `Authorize Increase`
+                    : `Authorize ${appAuthData.label}`
+                  : "Initiate Deauthorization"
+              }
+              maxTokenAmount={
+                isIncreaseAction ? maxAuthAmount : authorizedStake ?? "0"
+              }
+              placeholder={"Enter amount"}
+              minTokenAmount={
+                isIncreaseAction
+                  ? appAuthData.percentage === 0
+                    ? minAuthAmount
+                    : WeiPerEther.toString()
+                  : "0"
+              }
+              helperText={`Minimum ${formatTokenAmount(minAuthAmount)} T for ${
+                appAuthData.label
+              }`}
+            />
+          </GridItem>
+        )}
       </Grid>
+      {hasPendingDeauthorization && (
+        <>
+          <BodyMd>Pending Deauthorization</BodyMd>
+          <InfoBox
+            p="6"
+            direction={{ base: "column", sm: "row" }}
+            justifyContent="space-between"
+            alignItems="center"
+          >
+            <H5>
+              {formatTokenAmount(pendingAuthorizationDecrease)}{" "}
+              <BodyLg as="span">T</BodyLg>
+            </H5>
+            <Box minWidth="220px">
+              <>
+                <Progress
+                  h="2"
+                  borderRadius="md"
+                  colorScheme="brand"
+                  value={
+                    remainingAuthorizationDecreaseDelay === "0"
+                      ? 100
+                      : !isDeauthorizationReqestActive
+                      ? 0
+                      : calculatePercenteage(
+                          remainingAuthorizationDecreaseDelay,
+                          stakingAppAuthDecreaseDelay
+                        )
+                  }
+                />
+                <BodySm>
```

Comment:
Non-blocking:

This text is to close to the `Progress` component:
<img width="251" alt="image" src="https://user-images.githubusercontent.com/40306834/193017639-2025175e-3018-4d43-afd4-4918203b1144.png">

I would add some margin top to improve the look:

```suggestion
                <BodySm mt={"2"}>
```

Effect:

<img width="241" alt="image" src="https://user-images.githubusercontent.com/40306834/193017574-b0153c5b-a926-4a17-b262-8ec30780ed2b.png">

Reply:
29d49d4